applications/MeshingApplication/custom_utilities/simple_to_quadratic_tetrahedra_mesh_converter_utility.h

```diff
@@ -0,0 +1,347 @@
+// KRATOS  __  __ _____ ____  _   _ ___ _   _  ____
+//        |  \/  | ____/ ___|| | | |_ _| \ | |/ ___|
+//        | |\/| |  _| \___ \| |_| || ||  \| | |  _
+//        | |  | | |___ ___) |  _  || || |\  | |_| |
+//        |_|  |_|_____|____/|_| |_|___|_| \_|\____| APPLICATION
+//
+//  License:		 BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ariadna Cortés
+//
+
+#if !defined(KRATOS_TETRAHEDRA10_MESH_CONVERTER_UTILITY)
+#define  KRATOS_TETRAHEDRA10_MESH_CONVERTER_UTILITY
+
+// System includes
+
+/* Project includes */
+#include "custom_utilities/local_refine_tetrahedra_mesh.hpp"
+#include "geometries/tetrahedra_3d_10.h"
+#include "utilities/parallel_utilities.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+///@}
+///@name Type Definitions
+///@{
+///@}
+///@name  Enum's
+///@{
+///@}
+///@name  Functions
+///@{
+///@}
+///@name Kratos Classes
+///@{
+class Tetrahedra10MeshConverter : public LocalRefineTetrahedraMesh
```

Comment:
```suggestion
class SimpleToQuadraticTetrahedraMeshConverter : public LocalRefineTetrahedraMesh
```
Classes must be renamed accordingly. Also the test file.